Formula/qt5.rb

```diff
@@ -39,6 +39,19 @@ class Qt5 < Formula
     sha256 "48ff18be2f4050de7288bddbae7f47e949512ac4bcd126c2f504be2ac701158b"
   end
 
+  # Add missing install targets to Mac QtBase widget examples.  This fixes an
+  # issue with building from source with the `--with-examples` option set which
+  # was introduced between the releases of Qt versions 5.6.0 and 5.6.1 by
+  # <http://code.qt.io/cgit/qt/qtbase.git/commit/?id=5b5b7f97b833e9f91c41285718dc7b566cc37247>.
+  # (Remove this as soon as it is feasible and possible to do so!)
+  #
+  # Core formulae known to fail without this patch (as of 2016-06-19):
+  #   (None known.)
+  patch do
+    url "http://code.qt.io/cgit/qt/qtbase.git/patch/?id=58408ffa1b9c0b42a1719d3c8a4d4c62dec4fce6"
```

Comment:
My apologies:  I didn't know Qt's code repositories were available to browse via HTTPS.  I'll go ahead in fix that in my next commit, then.